Sources/SpeziOnboarding/ConsentView/ExportConfiguration+PDFPageFormat.swift

```diff
@@ -0,0 +1,28 @@
+//
+// This source file is part of the Stanford Spezi open-source project
+//
+// SPDX-FileCopyrightText: 2022 Stanford University and the project authors (see CONTRIBUTORS.md)
+//
+// SPDX-License-Identifier: MIT
+//
+
+import Foundation
+import TPPDF
+
+
+/// The ``ExportConfiguration`` enables developers to define the properties of the exported consent form.
+extension ConsentDocument.ExportConfiguration {
+    /// Returns a  `TPPDF.PDFPageFormat` which corresponds to Spezi's `ExportConfiguration.PaperSize`.
+    ///
+    /// - Parameters:
+    ///   - paperSize: The paperSize of an ExportConfiguration.
+    /// - Returns: A TPPDF `PDFPageFormat` according to the `ExportConfiguration.PaperSize`.
+    func getPDFPageFormat() -> PDFPageFormat {
+        switch paperSize {
```

Comment:
Wondering of we should move paper size into the `ConsentDocumentModel` in there as well?

Reply:
The paper size comes from the ExportConfiguration, and therefore is already stored inside the model.